### PR TITLE
(IRO-2696) Update upload file page.

### PR DESCRIPTION
**Pull request by azhurauski-sc**

### CODE REVIEW

**Review by ndemidovich-sc — APPROVED** [review]

File: src/routes/Onboarding/ImportAccount.tsx [hunk 1]
```diff
@@ -99,10 +111,26 @@ const ImportFileTab: FC<DesktopModeProps> = ({ desktopMode, onImport }) => {
...
+            onChange={e => {
+              setFile(e.target.files[0])
+            }}
           />
         </Button>
-      </Box>
+        {file && (
+          <Flex ml="34px" alignSelf="center" alignItems="center">
+            <FileIcon mr="8px" />
+            <h5>{file.name}</h5>
```
Comment (ndemidovich-sc) [inline src/routes/Onboarding/ImportAccount.tsx:122]: Please, think about long file name.

Comment (azhurauski-sc) [reply src/routes/Onboarding/ImportAccount.tsx:123]: @skylarrichard123 please check is current long file name view is ok

**Comment by skylarrichard123** [comment]
Screen Shot 2022-11-10 at 9 16 48 AM
After 32 character of a file name start truncating from the middle

**Comment by skylarrichard123** [comment]
actually what you have is good

**Review by skylarrichard123 — APPROVED** [review]

**Pull request merged**